Packs/GitLab/Integrations/GitLabEventCollector/GitLabEventCollector.py

```diff
@@ -74,6 +75,8 @@ def _iter_events(self):  # pragma: no cover
             response = self.call()
         except Exception as exc:
             demisto.info(f'Failed to get a response from the endpoint: {self.client.request.url}.\nError:\n{str(exc)}')
+            if '401' in str(exc):
```

Comment:
try checking for `if exc.res.status_code == 401` instead of checking for string contains.
And you need to catch DemistoException instead of Exception.

DemistoException contains the original response with the status_code